modules/configmap-create.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * builds/setting-up-trusted-ca
+
+[id="configmap-create-{context}"]
+= Creating a ConfigMap
+
+You can use the following command to create a `ConfigMap` from
+directories, specific files, or literal values.
+
+.Procedure
+
+* Create a ConfigMap:
+
+----
+$ oc create configmap <configmap_name> [options]
```

Comment:
fwiw there's a whole host of upstream docs:

https://kubectl.docs.kubernetes.io/pages/app_management/secrets_and_configmaps.html
https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#create-a-configmap

Reply:
Added to additional resources :)